1-fr-index.html

```diff
@@ -6,7 +6,7 @@
 ldjAppCategory: "Musique"
 ldjSlogan: "Jouez de la musique en ligne. Avec vos amis. Librement et gratuitement."
 ldjCreator: "Volker Fischer et contributeurs"
-metadescription: "Jamulus est un logiciel libre qui permet aux musiciens de jouer de la musique, de répéter ou simplement bœuffer en ligne et en temps réel via Internet."
+metadescription: "Jamulus est un logiciel à source ouverte qui permet aux musiciens de jouer de la musique, de répéter ou simplement bœuffer en ligne et en temps réel via Internet."
```

Comment:
See one of your comment about the translation of « open source » in another place on this doc.
https://github.com/jamulussoftware/jamuluswebsite/pull/47#discussion_r526877828
We can assume that « open source » is used here as in the body to design à « Free and open source software » that could be « logiciel libre » in french. We have no ambiguity in French compared to English with "free".
Don't forgot that the meta description should be used by search engine, so it is better to use « logiciel libre » here.

Reply:
In my comment you're mentioning above, the English strings was "free and open source" which can be summarized in French as "libre".

Here it's only written "open source" alone, so there is an ambiguity.

Then, it needs to be discussed against the English version first. I've opened #63 for this purpose.

Better to leave "source ouverte" here for 3.6.2, then we discuss and act in #63, then we'll make the needed changes (if approved) for 3.6.3 .